samples/rest/BingSpellCheckV7.py

```diff
@@ -1,53 +1,45 @@
-# Copyright (c) Microsoft Corporation. All rights reserved.
-# Licensed under the MIT License.
-
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.import json
```

Comment:
>import json [](http://example.com/codeflow?start=33&length=11)

Seems like you got an extra "import json" in all of these

Reply:
Done c1e833d
`BingImageSearchV7.py` had an extra `import os`